core/app/models/spree/taxon.rb

```diff
@@ -95,6 +98,23 @@ def child_index=(idx)
       move_to_child_with_index(parent, idx.to_i) unless new_record?
     end
 
+    def move_to_child_with_index(node, index)
+      # closure_tree does not provide any method that move nodes regarding position
+      # this method overrides move_to_child_with_index offered by awesome_nested_set
```

Comment:
@mamhoff you're right, not really overriding, it just follows the same logic as `awesome_nested_set`, which is totally removed.